apps/cookbook/src/app/examples/modal-example/modal-example-default.component.ts

```diff
@@ -208,6 +220,8 @@ export class ModalExampleDefaultComponent {
     const config: ModalConfig = {
       flavor,
       component: FirstEmbeddedModalExampleComponent,
+      interactWithBackground: this.interactWithBackground,
+      invertColors: this.invertColors,
```

Comment:
As per tech refinement, this should be handled by exposing css custom props for background + text color and let the target project set those